apps/dashboard/.env.example

```diff
@@ -12,20 +12,4 @@ SMTP_USER=
 SMTP_PASSWORD=
 
 # Unsplash setup - Optional if you want to use Unsplash images
-UNSPLASH_API_KEY=
-
-PUBLIC_TINYMCE_API_KEY=
-
-
-VITE_SENTRY_AUTH_TOKEN=
-VITE_SENTRY_ORG_NAME=
-VITE_SENTRY_PROJECT_NAME=
-
-# OpenAI setup - Optional if you want to use OpenAI
-OPENAI_API_KEY=
-
-# Sever URL - Optional if you want to use video upload and pdf downloads
-PUBLIC_SERVER_URL=
-
-# For getting metadata for signup users.
-PUBLIC_IP_REGISTRY_KEY=
```

Comment:
@R4Rohit23 There should be no change in this file.

It isn't related to the issue, please just copy the content of the file on main and paste it back here, then commit it. This is return this file to its original state